### PR TITLE
small readme updates

### PR DESCRIPTION
**Pull request by hxrts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,20 +22,18 @@ stack test
 
 ## How to run executables
 
-Running scripts and a REPL are the same thing - some scripts (in particular
-`rad/repl.rad`) just *define* a REPL.
+Executing a .rad script and running a REPL are equivalent. Some scripts (in particular
```
Comment (jkarni) [inline README.md]: I now see the original sentence was confusing, but I think it still is. Maybe:

```
The `radicle` executable accepts and interprets `.rad` scripts. Running a REPL is no different - a REPL is just a special type of script (such as `rad/repl.rad`) that implements REPL-like functionality.
```

File: README.md [hunk 2]
```diff
@@ -22,20 +22,18 @@ stack test
...
 
 ```
 stack exec client -- --config rad/repl.rad --url radicle.xyz
 ```
 
 ## The language
 
-The language currently lacking good documentation. A good place to get a sense
-for it is in the rad/ directory, which contains a number of radicle files.
+The language currently lacking complete documentation, however a good place to start is the rad/ directory, which contains a number of radicle files.
```
Comment (jkarni) [inline README.md]: There's now `docs.radicle.xyz`.

Comment (hxrts) [reply README.md]: ohh, nice. Also, maybe we should take a couple sentences from [here](http://oscoin.io/radicle.html) that give a little more context to what the repo is about?

**Comment by jkarni** [comment]
Closing since the README changed quite a bit, and most of this info is now on the website.

**Pull request closed without merging**